src/game.cpp

```diff
@@ -7685,8 +7685,12 @@ void game::examine( const tripoint &examp )
                 return;
             }
         }
+        if( mon != nullptr && mon->has_flag( MF_MILKABLE ) ) {
+            add_msg( m_info, _( "This is a milkable %s." ), mon->name().c_str());
+            iexamine::milk_source( mon );
```

Comment:
> Wouldn't that require to move all the pet menu and all that to monexamine.h/.cpp or monsterexamine.h/.cpp?

Not require, though it could include that.
You could leave the `pet_menu` in `game.cpp`, just have it refer to something in `monexamine` (one menu entry).

> Also as far as I know cows are "tamed" by way of manually selecting the most passive ones rather than more actively training(like a dog).

It should be safe to assume that all the cows that survived do fear anything that looks like a human.